src/content/docs/zh-tw/concepts/why-astro.mdx

```diff
@@ -1,45 +1,53 @@
 ---
 title: 為何選擇 Astro？
-description: Astro 是個多合一的網頁框架，用於打造快速且以內容為中心的網站。了解更多。
+description: "Astro 是個 web 框架，用於打造以內容為中心的網站，例如部落格、行銷、電商網站。進一步了解為何它是打造網站的絕佳選擇。"
 i18nReady: true
 ---
 
-Astro 是個**多合一**的**網頁框架**，用於打造**快速**且**以內容為中心**的網站。
+**Astro** 是個 web 框架，用於打造**以內容為中心的網站**，例如部落格、行銷、電商網站。Astro 以開創全新的[前端架構](/zh-tw/concepts/islands/)聞名，相較其他框架使用更少 JavaScript，複雜度更低。需要兼顧網站載入速度與 SEO 嗎？Astro 非常適合你。
 
-為何要在眾多網頁框架中選擇 Astro？以下的五個核心設計理念，有助於解釋為什麼我們要創造出 Astro、它解決了甚麼樣的問題以及為什麼 Astro 可能會是你專案或團隊的最佳選擇。
+## 特色
 
-#### Astro 是……
+Astro 是多合一的 web 框架，打造網站所需一應俱全。數以百計的[整合](https://astro.build/integrations/)和 [API hooks](/zh-tw/reference/integrations-reference/) 任君挑選，滿足客製需求。
 
-1. [以內容為主](#以內容為主)：Astro 是專為豐富內容網站而設計的。
-2. [伺服器優先](#伺服器優先)：網站藉由伺服器來渲染 HTML 在執行上會更加快速。
-3. [預設即迅速](#預設即迅速)：在 Astro 中不太可能會建立出緩慢的網站。
-4. [簡單易使用](#簡單易使用)：你不需要是個專家就能用 Astro 建立些東西。
-5. [靈活且齊全](#靈活且齊全)：超過 100 種 Astro 整合功能可供使用。
+以下是一些亮點：
 
+- **[群島](/zh-tw/concepts/islands/)**：以元件為基礎的 web 架構，專為以內容為中心的網站設計。
+- **[跨 UI 框架](/zh-tw/guides/framework-components/)**：支援 React、Preact、Svelte、Vue、Solid、Lit、HTMX、web component 等。
+- **[伺服器優先](/zh-tw/basics/rendering-modes/)**：避免在客戶端執行耗費資源的渲染。
+- **[預設零 JS](/zh-tw/basics/astro-components/)**：減少拖慢網站速度的客戶端 JavaScript。
+- **[內容合集](/zh-tw/guides/content-collections/)**：統一管理、驗證 Markdown 內容，同時提供 TypeScript 型別安全。
+- **[客製化](/zh-tw/guides/integrations-guide/)**：Tailwind、MDX 等數以百計的整合任君挑選。
 
-## 以內容為主
+## 設計理念
 
-**Astro 是專為豐富內容的網站設計的。** 這包括大部分的行銷網站、出版網站、文檔網站、部落格、作品集以及部分電子商務網站。
+為何要在眾多網頁框架中選擇 Astro 呢？以下五個核心設計理念，有助於解釋為什麼我們要打造 Astro、它解決了甚麼問題以及為什麼 Astro 可能是專案或團隊的最佳選擇。
```

Comment:
Missed `Astro is…` here?

Reply:
Thanks! Added here: https://github.com/withastro/docs/pull/7361/commits/2bef8099d74bb315493d67aca032bdc7056e54b3#diff-411f02195e31915769d02061af65a5c9167741c7dc87cd6d5d2fe0283ba84c5dR26